### DISCUSSION TITLE
環境清消機器人

### DISCUSSION THREAD

**Discussion by charlottechuang** [1101-GS4719-C/Python-Community-GS4719-C#337] [科技新知]
最近機場的那個本土案例搞得人心惶惶 
之後不知道會不會有什麼保全機器人之類的
把整個機場人員能淨空的淨空🥵 
https://news.ltn.com.tw/amp/news/health/breakingnews/3798563

**Comment by YuHengChen54** [comment]
如果機場跟防疫旅館的人員都換成機器人感覺就可以少很多傳染風險了，可能台灣現在還在+0~

**Reply by JAIMEHUNG** [reply]
感覺未來可以運用在人口流動的地方，還有醫院。對疫情來說很有幫助!

**Comment by 109302013evantsao** [comment]
這種機器人看起來效率應該還是不如真人來作的高，期待有一天這些機器人的效率能有更大的提升，且廣泛的應用

**Comment by vvvioletta** [comment]
科技業的工廠感覺也很需要 好像每次剛爆發都會有科技廠出事

**Comment by awkward-willy** [comment]
近期疫情相當嚴重
若能廣泛運用於醫院等公共場合
定能減少一定的感染率以及人力
集中投入患者照護的工作

**Comment by Lai27330** [comment]
有點像是進階版的掃地機器人會自己依照路徑前進，但能更精準的抵達需要清消的地方，對疫情防治蠻有幫助的。

**Comment by yenwei12138** [comment]
不知道實際使用上的清潔效果如何，但至少能減少清消人力需求

**Comment by treeann** [comment]
環境的維護是所有人的責任 加油!

**Comment by Yuhsin000** [comment]
有消毒功能的掃地機器人怎麼樣~

**Comment by stan890425** [comment]
像是這種固定路徑固定動作的都蠻適合機器人幫忙的

**Comment by yi-1111** [comment]
機場的人員都換成機器人，就可以少很多傳染風險

**Comment by Ben6uo** [comment]
雖然換成機械人會更好防疫 但同時現時機械人會有很多地方是清潔不了的

**Comment by sys511613** [comment]
用機器人可以解決清消人手不足的問題了

**Comment by leon10456** [comment]
感謝樓主分享 這感覺很棒!

**Comment by 110409538** [comment]
台威醫寶還可以進出電梯和上,下樓,這樣幾乎整個空間都可以交由他們來清潔.除了可以降低人力需求,他們即使使用較強效的清潔劑也不會有職業傷害,因為他們是機器人.

**Comment by liangyinglly** [comment]
用機器人比起可以解決清消人手不足的問題 更能解決人們對病毒的恐懼而不敢接近的問題

**Comment by HaoFish123** [comment]
感謝分享，希望這機器人可以更普及。

**Comment by ghost** [comment]
真希望把一些人渣順便清理一下><
